graphgym/contrib/loader/roland_btc.py

```diff
@@ -0,0 +1,182 @@
+"""
```

Comment:
We have merged `roland_xx` to a single file called `roland`, so I think this should be removed?